language_detector/main.py

```diff
@@ -1,8 +1,9 @@
-# -*- coding: utf-8 -*-
-
-"""This is the entry point of the program."""
-
 def detect_language(text, languages):
     """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    counts = { lang['name']:0 for lang in languages }
+    words = text.split()
+    for w in words:
+        for lang in languages:
+            if w in lang['common_words']:
+                counts[lang['name']]+=1
+    return max((count, name) for name, count in counts.items())[1]
```

Comment:
Oh, I just checked this to verify: The main difference between using get method vs using the usual way of getting stuff from a dict is that the usual way raises an exception if the relevant key isn't actually in the dict. With the get method, it returns a default value. The default value has a default value of None, but you can change that.

ie, `whatever.get('a key','NOPE')` returns `'NOPE'` if `'a key'` isn't in the dictionary.

`whatever['a key']` raises an exception if `'a key'` isn't in the dictionary